hw/ip/entropy_src/rtl/entropy_src_core.sv

```
@@ -2159,7 +2159,7 @@ module entropy_src_core import entropy_src_pkg::*; #(
   assign observe_fifo_thresh_met = fw_ov_mode && (observe_fifo_thresh <= sfifo_observe_depth);
 
   // fifo controls
-  assign sfifo_observe_push = fw_ov_mode && pfifo_postht_pop;
+  assign sfifo_observe_push = fw_ov_mode && pfifo_postht_pop && !sfifo_observe_full;
```

@martin-lueker I actually disagree with this change. This silently drops on the tail of the FIFO, of which I don't think was the agreed upon requirement. The design approach here is to pull from the FIFO before the FIFO gets to a full condition. This means that the Status bit needs to be set at a point where sw has enough time to empty the FIFO. For example, a threshold of 32 would allow 32 additional pushes (and more as pops occur) before the FIFO starts dropping on the input. Given the entropy speeds, this should be ample time. Likewise, if it is not, the threshold can be as low a 1, giving 63 time slots of delay available.